### PR TITLE
Davical 1.1.8 version bump

### PR DESCRIPTION
**Pull request by tillschaefer**

### CODE REVIEW

**Comment by orlitzky** [comment]
The AWL bump looks good to me, but I'm not qualified to comment on the webapp stuff.

**Comment by orlitzky** [comment]
The others I believe, but why was `php[cgi]` added as a dependency?

**Comment by tillschaefer** [comment]
> The others I believe, but why was `php[cgi]` added as a dependency?
I was following their wiki page: https://wiki.davical.org/index.php/DAViCal_Dependencies

After looking in the code, it seems to me that the cgi dependency is optional. Furthermore, I have used davical without cgi for years now. Do you think it might be better to drop the dependency?

**Comment by orlitzky** [comment]
Ah ok, I didn't know about that page. I'm not 100% certain what php-cgi does on Debian, but I think what it's saying amounts to "you need some kind of web interface to PHP." On Gentoo, USE=cgi only gives you one of three possible web interfaces to PHP, namely CGI. But the other two, php-fpm and mod_php, both work equally well for any web application. So really, what you want is to say "php[apache] or php[cgi] or php[fpm]".

And you're in luck: that's exactly what the `virtual/httpd-php` dependency does. So I think you can drop `php[cgi]`, because we've got it covered already.

**Comment by tillschaefer** [comment]
> So I think you can drop `php[cgi]`, because we've got it covered already.
Done

**Comment by tillschaefer** [comment]
Any progress here? Anything else, that blocks the request?

File: www-apps/davical/davical-1.1.7-r2.ebuild [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
-HOMEPAGE="https://davical.org/"
+HOMEPAGE="https://www.davical.org/"
 SRC_URI="https://www.davical.org/downloads/${PN}_${PV}.orig.tar.xz -> ${P}.tar.xz"
 
 LICENSE="GPL-2+"
 KEYWORDS="~amd64 ~x86"
+IUSE="ldap"
 
-DEPEND=">=dev-php/awl-0.59
-	sys-devel/gettext"
```
Comment (mgorny) [inline www-apps/davical/davical-1.1.7-r2.ebuild]: You sure you don't need this? Then how are the locales being built?

**Comment by mgorny** [comment]
Ping.

**Comment by mgorny** [comment]
Closing due to no activity. Feel free to reopen once you are available again.

**Comment by tillschaefer** [comment]
I am back. Sorry taking so long. The updated version of ebuild is already pushed. 

Can someone with more permissions reopen this pull request for me?

**Pull request closed without merging**